consensus/istanbul/backend/announce.md

```diff
@@ -0,0 +1,140 @@
+# Celo Announce Protocol
+
+While here referred as a protocol by itself, Announce is technically a subset of the Istanbul protocol, on the [RLPx] transport, that facilitates Validators to privately discover the eNodeURL of other Validators in the network, using the underlying p2p eth network. This is used by Istanbul to have direct connections between Validators, speeding up the consensus phase of block generation.
```

Comment:
Even though now it is a subset of the Istanbul protocol, I think it makes sense for it to be separated into it's own protocol. You could consider it a parallel of the discovery protocol (which is separate to the eth protocol), except that rather than simply discovering nodes it discovers validators.

Reply:
I think its fine as is, right now announce _is_ part of the istanbul protocol, I was just airing the idea that I think eventually I think it would make more sense for this protocol to be entirely separate.